src/client/diagnosticUtil.ts

```diff
@@ -66,10 +45,10 @@ export class DiagnosticUtil {
       }
 
       const matches = new RegExp(/(\[Line: (\d+), Col: (\d+)] )?(.*)/).exec(problem);
-      if (matches && matches[2] && matches[3] && matches[4]) {
+      if (matches?.[2] && matches[3] && matches[4]) {
```

Comment:
I'm having to stare at this to ensure I know what it's checking...basically we're verifying that there were at least 4 results from the regex & none or empty strings correct?

Reply:
I never understood this part, either.  I've never touched this code (linter autofix for optional-chain)